proc-macros/src/render_server.rs

```diff
@@ -147,25 +150,23 @@ impl RpcDescription {
 							#into_response::into_response(context.as_ref().#rust_method_name(connection_details, #params_seq).await)
 						})
 					})
-				} else {
-					if method.signature.sig.asyncness.is_some() {
-						handle_register_result(quote! {
-							rpc.register_async_method(#rpc_method_name, |params, context| async move {
-								#parsing
-								#into_response::into_response(context.as_ref().#rust_method_name(#params_seq).await)
-							})
+				} else if method.signature.sig.asyncness.is_some() {
```

Comment:
I suppose this is unrelated and just fixes a clippy warning?

Reply:
Yes, I can revert if this is not the right place to do this but CI tests linted this so I fixed it.